tests/test_EN_US.py

```diff
@@ -24,7 +24,16 @@ def test_en_us(self):
             "twenty-three hundred thousand four",
             "sixty-eight billion, two hundred two million and two",
             "sixty eight billion two hundred two million two",
-            "nine trillion eight hundred seven")
+            "nine trillion eight hundred seven",
+            "ninety nine point nine",
+            "ninety nine point nine nine nine nine nine nine nine six",
+            "fifteen point four",
+            "two thousand point five",
+            "eleven million point three three",
+            "one billion point six four",
+            "six hundred point nine nine",
+            "one point zero zero four",
```

Comment:
Do you want to support "oh" as a synonym for "zero"?

Reply:
Yeah sure